### PR TITLE
fix resume in train.py for ssd

### PR DESCRIPTION
**Pull request by zhangwenwen**

### CODE REVIEW

**Comment by Hakuyume** [comment]
Why do you want to change that line?
We should resume `trainer` because it contains the number of proceeded iterations, status of the optimizer and learning rate.
